exporter/otlp/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History: opentelemetry-exporter-otlp
 
+### v0.22.0 / 2022-05-18
+
+* ADDED: Otlp grpc 
```

Comment:
wait does this mean the OTLP gRPC gem needs to be released? 😕

Reply:
We can rework the changelog message to signify no recent changes.